modules/mobile-agents/windows/jax-rs/src/main/java/org/wso2/carbon/mdm/mobileservices/windows/operations/util/OperationReply.java

```diff
@@ -639,19 +643,19 @@ public SequenceTag buildSequence(Operation operation, SequenceTag sequenceElemen
         return monitorItems;
     }
 
-    public List<Item> buildDeleteInfo(Operation operation) {
+    public List<Item> buildDeletePassCode(Operation operation) {
         List<Item> deleteItems = new ArrayList<>();
         Item deleteItem = new Item();
         Target target = new Target();
-        String operationCode = operation.getCode();
         if (operation.getCode().equals(PluginConstants.OperationCodes.PASSCODE_POLICY)) {
```

Comment:
possible NPE due to checking an unknown value against a constant. should be the other way around.